src/derivatives/imaging.md

```diff
@@ -153,10 +153,9 @@ Template:
 
 A binary (1 - inside, 0 - outside) mask in the space defined by the [`space` entity](../appendices/entities.md#space).
 If no transformation has taken place, the value of `space` SHOULD be set to `orig`.
-If the mask is an ROI mask derived from an atlas, then the [`label` entity](../appendices/entities.md#label)) SHOULD
-be used to specify the masked structure
-(see [Common image-derived labels](#common-image-derived-labels)),
-and the `Atlas` metadata SHOULD be defined.
+If the mask is an ROI mask derived from an atlas segmentation,
```

Comment:
Not sure if there's a more extended discussion required here regarding "atlas" vs. "template" or "brain" vs. "structure".

Say one derives a brain mask by registering to a template image and then applying the inverse transformation to a brain mask defined with respect to the template image. This is an entirely valid ROI mask. Two issues however:

1.  The main precedent I'm aware of, fMRIPrep, uses `_desc-brain` rather than `_label-brain`.
2.  In my mind, there was no "atlas" used in this process. An "atlas", to me, requires a plurality of labels.

Reply:
I don't see any problems with this passage.  @Lestropie, I don't understand your example: 
> Say one derives a brain mask by registering to a template image and then applying the inverse transformation to a brain mask defined with respect to the template image. This is an entirely valid ROI mask.

This example entirely refers to a "brain mask" which to me is *not* an ROI, but an image defining brain/non-brain.  Hence I would never call the input or output a "ROI mask". 

I may be out of sync with this PR, but to me "ROI" implies a sub-"region" and excludes a brain mask.

Reply:
If it's sufficiently unambiguous and unanimous that "ROI" = "subset of brain", then that is true. However I'm not 100% convinced that's the case; I don't see why the brain couldn't be considered a "region" of interest with respect to the whole image, just as could be eg. the skull or face. 

If yours were to be an accepted definition, specifically in the context of a *brain* imaging data structure, especially if it were to instruct that different entities be used for ROI masks vs. not-ROI masks, perhaps that necessitates inclusion in the definition dictionary. The only relevant reference I can find is the "`Type`" metadata field, which lists "`Brain`" and "`ROI`" as being two separate things, but is fairly hidden away.

-----

To put more explicitly the consequence of the point: is there a reason why the `_label-` entity "SHOULD" doesn't just apply to `_mask.` suffix data, regardless of whether it is a "ROI" derived from an "atlas" (over and above the potential ambiguity of those two terms)?

Reply:
I think this thread may be getting a bit off-topic. The goal of this particular change is simply to use the phrase "atlas segmentation" instead of "atlas". If you would like to make a clarification that is consistent with the existing definition (modulo the removal of this narrowly defined atlas), that would be in-scope for this PR.

If you think the existing text is insufficiently clear, then I would respectfully ask that we postpone that discussion until this one is complete, or at least move it to a new issue/PR.